### PR TITLE
(DOCSP-21720) Adds --skipConfig option to Connect page

### PR DESCRIPTION
**Pull request by sarahsimpers**
[Build Link](https://workerpool-boxgs.mongodbstitch.com/pages/job.html?collName=queue&jobId=6245ad152e8f69777c555b9d)
[Jira Ticket Link](https://jira.mongodb.org/browse/DOCSP-21720)
[Staging](https://docs-mongodbcom-staging.corp.mongodb.com/atlas-cli/docsworker-xlarge/DOCSP-21720/connect-atlas-cli/#follow-these-steps)

### CODE REVIEW

File: source/connect-atlas-cli.txt [hunk 1]
```diff
@@ -113,7 +113,17 @@ from the {+atlas-cli+}.
...
+
+            Run the ``atlas auth login`` command in your terminal with the flag
+            to skip profile setup.
+
+            .. code-block:: sh
+
+               atlas auth login --skipConfig
+
+            The command opens a browser window and returns a one-time
+            activation code. This code expires after 10 minutes.
```
Comment (andreaangiolillo) [inline source/connect-atlas-cli.txt:126]: mh... I have the impression that we are suggesting running the login with `skipConfig`. I would keep the example without the flag and then add a sub-step saying that they can use `skipConfig` if they don't want to set the org/proj Ids

**Review by JuliaMongo — APPROVED** [review]
LGTM, with Andrea's comment.

File: source/connect-atlas-cli.txt (hunk 1 above)
Comment (sarahsimpers) [reply source/connect-atlas-cli.txt:126]: @andreaangiolillo Currently we have two step-by-step procedures for logging in: [Connect with minimum required settings ](https://docs-mongodbcom-staging.corp.mongodb.com/atlas-cli/docsworker-xlarge/DOCSP-21720/connect-atlas-cli/#connect-with-minimum-required-settings)and [Create a profile](https://docs-mongodbcom-staging.corp.mongodb.com/atlas-cli/docsworker-xlarge/DOCSP-21720/atlas-cli-save-connection-settings/#create-a-profile). This was by design to give users an option to log in quickly without spending time setting up a profile at all if they chose to do so. Using --skipConfig would be the minimum required settings unless this is not allowed for first-time login, so I think it's appropriate to have this as the command on this page. We do specify on [the tabs here](https://docs-mongodbcom-staging.corp.mongodb.com/atlas-cli/docsworker-xlarge/DOCSP-21720/connect-atlas-cli/#select-a-connection-method) that they have both options for login.

Comment (andreaangiolillo) [reply source/connect-atlas-cli.txt:126]: Ack. Thanks for the clarification!

**Review by andreaangiolillo — APPROVED** [review]

**Pull request merged**